tests/test_help_plugin.py

```
@@ -391,6 +410,37 @@ async def run_test():
+        mock_load_plugins.return_value = []
+        self.plugin.matches = MagicMock(return_value=True)
+        self.plugin.send_matrix_message.side_effect = RuntimeError("send failed")
+
+        room = MagicMock()
+        room.room_id = "!test:matrix.org"
+        full_message = "!help"
+        event = MagicMock()
+        event.body = full_message
+        event.source = {"content": {"formatted_body": ""}}
+
+        async def run_test():
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `run_test`

Add return type annotation: `None`

(ANN202)